src/app/code/community/Omise/Gateway/Model/OmiseTransfer.php

```diff
@@ -5,13 +5,30 @@ class Omise_Gateway_Model_OmiseTransfer extends Omise_Gateway_Model_Omise
      * @param string $id
      * @return OmiseTransfer|array
      */
-    public function retrieveOmiseTransfer($id = '')
+    public function retrieveOmiseTransfer($params)
     {
+        if(!isset($params['page']))
+            $params['page'] = 1;
+
+        if(!isset($params['limit']))
+            $params['limit'] = 10;
+
         try {
-            return OmiseTransfer::retrieve('', $this->_public_key, $this->_secret_key);
+
+            $check = OmiseTransfer::retrieve('', $this->_public_key, $this->_secret_key);
```

Comment:
- I didn't get 503 error in index.php/admin/omise/charges url when open dashboard page without charge history. I instead get this {"from":"1970-01-01T00:00:00+00:00","to":"2015-08-17T21:12:52+00:00","offset":0,"limit":20,"total":0,"data":[]}. even i input wrong api key, i got this result {"from":null,"to":null,"offset":null,"limit":null,"total":null,"data":null}
- I fixed live mode
- I added ?limit=1 in both charge and transfer model